src/DssSpell.sol

```diff
@@ -58,311 +68,102 @@ contract DssSpellAction is DssAction {
-        // VOTE: https://vote.makerdao.com/polling/Qmbndmkr#vote-breakdown
-
-        // VOTE: https://vote.makerdao.com/polling/QmRxNdG7
-        // Phoenix Labs - 347,100 DAI - 0x115F76A98C2268DaE6c1421eb6B08e4e1dF525dA
-        DssExecLib.sendPaymentFromSurplusBuffer(PHOENIX_LABS_2, 347_100);
-
-        // VOTE: https://vote.makerdao.com/polling/QmV9MR8B
-        // Viridian Protector Advisory Company - 257,250 DAI - 0xA1E62c6321eEd0ECFcF2f382c8c82FD940D83c07
-        DssExecLib.sendPaymentFromSurplusBuffer(VIRIDIAN_TRANSFER,   257_250);
-
-        // ----- Additional ESM authorization fix -----
-        DssExecLib.authorize(RWA_LIQ_ORACLE, ESM);
+        uint256 lineReduction;
+        uint256 line;
+
+        // ---------- RWA008-A Offboarding ----------
+        // Poll: N/A
+        // Forum: https://forum.makerdao.com/t/security-tokens-refinancing-mip6-application-for-ofh-tokens/10605/51
+
+        // Set RWA008-A Debt Ceiling to 0
+        (,,,line,) = vat.ilks("RWA008-A");
+        lineReduction += line;
+        DssExecLib.setIlkDebtCeiling("RWA008-A", 0);
+
+        // ---------- First Stage of Offboarding ----------
+        // Poll: https://vote.makerdao.com/polling/QmPwHhLT
+        // Forum: https://forum.makerdao.com/t/decentralized-collateral-scope-parameter-changes-1-april-2023/20302
+
+        // Set YFI-A line to 0
+        (,,,line,) = vat.ilks("YFI-A");
+        lineReduction += line;
+        DssExecLib.removeIlkFromAutoLine("YFI-A");
+        DssExecLib.setIlkDebtCeiling("YFI-A", 0);
+
+        // Set MATIC-A line to 0
+        (,,,line,) = vat.ilks("MATIC-A");
+        lineReduction += line;
+        DssExecLib.removeIlkFromAutoLine("MATIC-A");
+        DssExecLib.setIlkDebtCeiling("MATIC-A", 0);
+
+        // Set LINK-A line to 0
+        (,,,line,) = vat.ilks("LINK-A");
+        lineReduction += line;
+        DssExecLib.removeIlkFromAutoLine("LINK-A");
+        DssExecLib.setIlkDebtCeiling("LINK-A", 0);
+
+        // Decrease Global Debt Ceiling in accordance with Offboarded Ilks
+        vat.file("Line", vat.Line() - lineReduction);
+
+        // ---------- Stability Fee Changes ----------
+        // Poll: N/A
+        // Forum: https://forum.makerdao.com/t/decentralized-collateral-scope-parameter-changes-1-april-2023/20302
+
+        // Increase the WBTC-A Stability Fee from 1.75% to 4.90%
+        DssExecLib.setIlkStabilityFee("WBTC-A", FOUR_NINE_PCT_RATE, /* doDrip = */ true);
+
+        // Increase the WBTC-B Stability Fee from 3.25% to 4.90%
+        DssExecLib.setIlkStabilityFee("WBTC-B", FOUR_NINE_PCT_RATE, /* doDrip = */ true);
+
+        // Increase the WBTC-C Stability Fee from 1.00% to 4.90%
+        DssExecLib.setIlkStabilityFee("WBTC-C", FOUR_NINE_PCT_RATE, /* doDrip = */ true);
+
+        // Increase the GNO-A Stability Fee from 2.50% to 4.90%
+        DssExecLib.setIlkStabilityFee("GNO-A",  FOUR_NINE_PCT_RATE, /* doDrip = */ true);
+
+        // ---------- PE MKR Vesting Stream Cleanup ----------
```

Comment:
Should probably decrease the allowance to the vest contract, but I don't really mind either way.

Reply:
Nevermind I'm being told this was intentionally omitted.